jsonnet/kube-prometheus/components/prometheus-adapter.libsonnet

```diff
@@ -27,8 +27,8 @@ local defaults = {
   config: {
     resourceRules: {
       cpu: {
-        containerQuery: 'sum(irate(container_cpu_usage_seconds_total{<<.LabelMatchers>>,container!="",pod!=""}[5m])) by (<<.GroupBy>>)',
```

Comment:
nit: Maybe we should surface those range parameters in `defaults` and use string extrapolation here? Sth like:

```
rangeIntervals:
  kubelet: 120
  nodeExporter: 120
  windowsExporter: 120
```

This would open a way to extract scrape_intervals from ServiceMonitors from the component, multiply by 4 and have an automatic range which would conform to prometheus best practices while presenting meaningful data.

Reply:
good idea :+1:

Reply:
I made the change, please have a new look :)